packages/@react-stately/dnd/src/useDroppableCollectionState.ts

```diff
@@ -11,10 +11,18 @@
  */
 
 import {Collection, DragTypes, DropOperation, DroppableCollectionProps, DropTarget, ItemDropTarget, Node} from '@react-types/shared';
+import {Key, useCallback, useRef, useState} from 'react';
 import {MultipleSelectionManager} from '@react-stately/selection';
-import {useRef, useState} from 'react';
 
-export interface DroppableCollectionStateOptions extends DroppableCollectionProps {
+interface DropOperationEvent {
+  target: DropTarget,
+  types: DragTypes,
+  allowedOperations: DropOperation[],
+  isInternalDrop: boolean,
```

Comment:
I'd like to rename this to just `isInternal` across all apis. IMO the drop part is a little confusing in APIs where you haven't dropped yet (e.g. getDropOperation).

Reply:
makes sense, will update